item-impl/src/main/java/com/example/auction/item/impl/PItemEntity.java

```diff
@@ -103,14 +108,26 @@ private Behavior cancelled(PItemState state) {
         builder.setReadOnlyCommandHandler(GetItem.class, this::getItem);
 
         // Ignore these commands, they may come due to at least once messaging
-        builder.setReadOnlyCommandHandler(UpdatePrice.class,
-                (cmd, ctx) -> ctx.reply(Done.getInstance()));
-        builder.setReadOnlyCommandHandler(FinishAuction.class,
-                (cmd, ctx) -> ctx.reply(Done.getInstance()));
+        builder.setReadOnlyCommandHandler(UpdatePrice.class, this::alreadyDone);
+        builder.setReadOnlyCommandHandler(FinishAuction.class, this::alreadyDone);
 
         return builder.build();
     }
 
+    /**
+     * Convenience method to handle when a command has already been processed (idempotent processing).
+     */
+    private void alreadyDone(Object command, ReadOnlyCommandContext<Done> ctx) {
```

Comment:
I suggested "ignore" because that's the word you used in the comments.

To be honest, I'm now kind of confused about when it is necessary to send a reply. Is it always necessary? Then why doesn't the API enforce this, and what is the use of the existing `ctx.done` method?